src/compiler.c

```diff
@@ -3735,7 +3739,7 @@ static void CompIf(Stat stat)
     /* print a comment                                                     */
     if ( CompPass == 2 ) {
         Emit( "\n/* if " );
-        PrintExpr(READ_EXPR(stat, 0));
```

Comment:
Why is this line (for example) READ_EXPR, when other lines are READ_EXPR_IN_BODY?

Reply:
I was confused by that comment as that line actually is removed :-). But I assume you meant the next line, so I'll comment there.